code/src/App.js

```diff
@@ -1,9 +1,138 @@
-import React from 'react'
+import React, { useState } from "react"
+
+import "./style.css"
+import { Summary } from "./components/Summary.js"
+import { Header } from "./components/Header.js"
 
 export const App = () => {
+  const [reflection, setReflection] = useState(""); // the thing we have in the paranthesis is the initial state. If we console log status it's gonna show the thing in the paranthesis. If it's gonna show strings you use "" if it's gonna show bolean you write true or false etc, if i's gonna show an array you write []. 
+  const [reason, setReason] = useState("");
+  const [weather, setWeather] = useState("");
+  const [culture, setCulture] = useState ("");
+  const [showSummary, setShowSummary] = useState (false);
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowSummary(true)
+  }
+
+  const date = () => {
+    const day = new Date();
+    const local = day.toLocaleDateString('en-GB');
+    document.getElementById("demo").innerHTML = local;
+  }
+  console.log(date)
+
+  const event = new Date(Date.UTC('en-GB'));
+  const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
```

Comment:
Sweet, did you mean to display the current date/time for the user to see when filling out the app? Would have loved to see this in action!✨